.github/workflows/release_master_fontpatcher-zip.yaml

```diff
@@ -0,0 +1,45 @@
+name: Release font patcher from master
+
+on:
+  create:
+  push:
+    branches: master
+    paths:
+      - font-patcher
+      - src/glyphs/**
+      - src/archive-font-patcher-readme.me
+      - bin/scripts/archive-font-patcher.sh
+      - bin/scripts/name_parser
+      - .github/workflows/release_master_fontpatcher-zip.yaml
+  workflow_dispatch:
+  
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    
+    steps:
+    
+    - name: Grab the script and its dependencies from the repo
+      uses: snow-actions/sparse-checkout@v1.2.0
+      with:
+        patterns: |
+          font-patcher
+          src/glyphs
+          src/archive-font-patcher-readme.md
+          bin/scripts/archive-font-patcher.sh
+          bin/scripts/name_parser
+          
+    - run: chmod +x 
+        font-patcher
+        bin/scripts/archive-font-patcher.sh
+        bin/scripts/name_parser        
```

Comment:
I guess the action does not restore the access rights?

If you want to `+x` the scripts in `name_parse` is not some `-R` option or a glob (`*`) needed for `chmod`?

Reply:
I think this was a mistake on my part for something else — I have to double check, but I think a different action I used previously to clone the repo used tarballs under the hood and didn’t set permissions properly.

We should be able to just remove the line and not have a problem:

This checkout is just a real sparse checkout (it’s great, I _love_ sparse checkout, it’s the only way to wrangle big repos like https://GitHub.com/QMK/qmk) so it shouldn’t mess with the permissions.